BREAKING.md

```diff
@@ -113,6 +115,19 @@ body {
 
 This allows components to inherit the color properly when used outside of Ionic Framework and is required for custom themes to work properly. However, it may have unintentional side effects in apps if the color was not expected to inherit.
 
+<h4 id="version-8x-dynamic-font">Dynamic Font</h4>
+
+The `core.css` file has been updated to enable dynamic font scaling by default:
+
+```diff
+html {
+- --ion-default-dynamic-font: -apple-system-body;
++ --ion-dynamic-font: -apple-system-body;
+}
+```
+
+The `--ion-default-dynamic-font` variable has been removed and replaced with `--ion-dynamic-font`. This change may have unintentional side effects in apps if the font was not expected to scale dynamically.
```

Comment:
I think we need to include something for users that were enabling the dynamic font. In our [guide for enabling dynamic font scaling](https://ionicframework.com/docs/layout/dynamic-font-scaling#enabling-dynamic-font-scaling-in-ionic) we tell the developers how to enable it. Maybe we should say somewhere that developers can remove this code and it will still work? 

I am just not sure how showing the code snippet above will be helpful to developers without seeing how `--ion-dynamic-font` is used or linking to what dynamic font is. We could show them how to disable it, or just let them know why they should keep it enabled and that they shouldn't be changing the `font-size` of their apps on the `html` element.

Reply:
Hopefully this sounds better.